From 7739b0181040cee5e8ddd0e143a35d10d1b8429b Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Fri, 7 Aug 2020 23:34:37 +0000
Subject: [PATCH] CMakeLists.txt: install the library as well

* otherwise controller_node binary is linked against unavailable libomnibase_control.so
  ERROR: omnibase-control-0.0.2-1-r0 do_package_qa: QA Issue: /usr/opt/ros/melodic/lib/omnibase_control/controller_node contained in package omnibase-control requires libomnibase_control.so, but no providers found in RDEPENDS_omnibase-control? [file-rdeps]

Upstream-Status: Pending

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7f6c406..f392c1a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,3 +42,4 @@ target_link_libraries(teleop_node ${PROJECT_NAME} ${catkin_LIBRARIES})
 
 install(TARGETS controller_node RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 install(TARGETS teleop_node RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+install(TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${CATKIN_PACKAGE_LIBRARY_DESTINATION})
